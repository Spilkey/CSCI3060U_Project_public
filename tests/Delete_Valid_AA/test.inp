loginuser01deleteuser05logout