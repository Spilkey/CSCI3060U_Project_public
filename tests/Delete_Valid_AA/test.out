02 user05          FS 000000.00
00 user01          AA 000000.00